Extensions/TileMap/JsExtension.js

```diff
@@ -644,6 +644,10 @@ module.exports = {
       if (this._instance.hasCustomSize()) {
         this._pixiObject.width = this._instance.getCustomWidth();
         this._pixiObject.height = this._instance.getCustomHeight();
+        this._pixiObject.scale.x =
```

Comment:
I added this bit to fix the tilemap's Y position jumping down when changing the width of one of the tilemaps.
This fixed it in the IDE, but its still broken on the runtime. Custom size messes up their position in the runtime

Reply:
I suspect its caused by using this._pixiObject to set scale, width and height. Pixi-tilemap is likely overriding those with crazy values

Reply:
The new code in the IDE seems fine. Note that you should remove the two lines above:
```
 this._pixiObject.width = this._instance.getCustomWidth();
 this._pixiObject.height = this._instance.getCustomHeight();
```

because they are not useless (setting the scale x and y will change the width/height) (the width of a pixi object is simply `original width * scaleX`. When you change the width, Pixi actually changes the scale).